infra/feast-operator/internal/controller/services/util.go

```diff
@@ -160,8 +166,23 @@ func ApplyDefaultsToStatus(cr *feastdevv1alpha1.FeatureStore) {
 
 func setServiceDefaultConfigs(defaultConfigs *feastdevv1alpha1.DefaultConfigs) {
 	if defaultConfigs.Image == nil {
-		defaultConfigs.Image = &DefaultImage
+		img := getFeatureServerImage()
+		defaultConfigs.Image = &img
+	}
+}
+
+func getFeatureServerImage() string {
+	if img, exists := os.LookupEnv("RELATED_IMAGE_FEATURE_SERVER"); exists {
+		return img
 	}
+	return DefaultImage
+}
+
+func getGrpcCurlImage() string {
```

Comment:
Just curious why is this image needed? where was this being used?

Reply:
`disableInitContainers` adds ability to disable the `feast repo initialization' and the 'grpc healthcheck' initContainers if the user chooses to do so.